packages/app-blueprint/files/config/ember-cli-update.json

```diff
@@ -2,13 +2,11 @@
   "schemaVersion": "1.0.0",
   "packages": [
     {
-      "name": "ember-cli",
+      "name": "<%= blueprint %>",
       "version": "<%= blueprintVersion %>",
       "blueprints": [
         {
           "name": "<%= blueprint %>",
-          "outputRepo": "https://github.com/ember-cli/ember-<%= blueprint === 'app' ? 'new' : 'addon' %>-output",
-          "codemodsSource": "ember-<%= blueprint %>-codemods-manifest@1",
```

Comment:
We probably do want a codemodsSource? This removes it for classic as well and so I don't believe ember-cli-update's run codemods will work without this. 

They are kept here in a monorepo https://github.com/ember-cli/ember-cli-update-codemods-manifest/